my-app/src/components/BlogItem.tsx

```diff
@@ -1,47 +1,47 @@
-import React, {useEffect, useState} from 'react'
+import React, {useContext} from 'react'
 import {Link} from 'react-router-dom'
-import {Chip} from '@mui/material'
+import {Chip, IconButton} from '@mui/material'
 import face from '../images/photo.png'
 import styled from 'styled-components'
-import cover from '../images/ddl.jpg'
+import {BlogPostType, PostContext} from "../pages/homePage/BlogHomeContext";
 
-export type BlogType = {
-    _id: string
-    title: string
-    text: string
-    createdAt: Date
-    user: any
-    imageUrl:string
-}
 
 export type PostType = {
-    post: BlogType
+    post: BlogPostType
 }
 
 
 export const BlogItem = (props: PostType) => {
-    const [data, setData] = useState<BlogType>()
-    useEffect(()=>{
-        setData(props.post)
-    },[])
-   console.log(data?.user)
+
+    const logic = useContext(PostContext)
     return (
         <Container>
-            <ItemWrapper src={`http://localhost:3222${data?.imageUrl}`} alt='cover'/>
-            <H3Wrapper>{data?.title}</H3Wrapper>
-            <H3Wrapper>{data?.text}</H3Wrapper>
-            <H3Wrapper>{data?.user['fullName']}</H3Wrapper>
-            {/*<H3Wrapper>{`http://localhost:3222${data?.imageUrl}`}</H3Wrapper>*/}
-
-
+            <ItemWrapper src={`http://localhost:3222${props.post.imageUrl}`} alt='cover'/>
+            <H3Wrapper>{props.post.title}</H3Wrapper>

```

Comment:
use destruction for props. 
`const { post }  = props`

Reply:
done